src/cavendish_particle_tracks/_magnification_dialog.py

```diff
@@ -31,6 +31,16 @@ def __init__(self, parent=None):
         self.b2 = Fiducial()
 
         # region UI Setup
+        self.ui_setup()
+        self.cal_layer = self.parent.viewer.add_points(
+            name="Points_Calibration"
+        )
+
+        @self.cal_layer.mouse_drag_callbacks.append
```

Comment:
@palvarezc It's really ugly, but this is the only way I've managed to get callbacks to work. Intention is that by now being able to use callbacks, it might be lot easier to implement what we're trying to do.

Reply:
Might be worth making this a separate pull request?

Reply:
Think you're right. This PR has ballooned. Will make these improvements on a different PR so that it's implemented into the current workflow and can at least be tested.